src/CSSList/CSSBlockList.php

```diff
@@ -96,42 +96,42 @@ protected function allValues(
      */
     protected function allSelectors(array &$result, $specificitySearch = null): void
     {
-        /** @var array<int, DeclarationBlock> $aDeclarationBlocks */
-        $aDeclarationBlocks = [];
-        $this->allDeclarationBlocks($aDeclarationBlocks);
-        foreach ($aDeclarationBlocks as $oBlock) {
+        /** @var array<int, DeclarationBlock> $declarationBlocks */
+        $declarationBlocks = [];
+        $this->allDeclarationBlocks($declarationBlocks);
+        foreach ($declarationBlocks as $oBlock) {
             foreach ($oBlock->getSelectors() as $selector) {
                 if ($specificitySearch === null) {
                     $result[] = $selector;
                 } else {
-                    $sComparator = '===';
-                    $aSpecificitySearch = \explode(' ', $specificitySearch);
-                    $iTargetSpecificity = $aSpecificitySearch[0];
-                    if (\count($aSpecificitySearch) > 1) {
-                        $sComparator = $aSpecificitySearch[0];
-                        $iTargetSpecificity = $aSpecificitySearch[1];
+                    $comparator = '===';
+                    $specificitySearchResults = \explode(' ', $specificitySearch);
```

Comment:
I think this should be named with something like 'parts', 'elements' or 'components'.  The input string contains either a number (e.g. "1"), or an operator and a number (e.g. ">= 1").  The result goes into `$result`.

Reply:
Maybe `$expressionParts`?

Reply:
Yes I think that works.  It's only used locally in that small section of code.